pkgs/by-name/ei/eiffelstudio/package.nix

```diff
@@ -0,0 +1,125 @@
+{
+  copyDesktopItems,
+  fetchurl,
+  lib,
+  makeDesktopItem,
+  makeWrapper,
+  pkgs,
+  stdenv,
+
+  dotnetSupport ? false, # Untested integration with .NET, disabled for now
+}:
+
+let
+  platformName =
+    {
+      "x86_64-linux" = "linux-x86-64";
+      "aarch64-linux" = "linux-arm64";
+      "x86_64-darwin" = "macosx-x86-64"; # Doesn't compile with clang yet
+      "aarch64-darwin" = "macosx-armv6"; # Need to do more testing on macOS
+    }
+    .${stdenv.system} or null;
+in
+stdenv.mkDerivation rec {
+  pname = "eiffelstudio";
+  version = "25.02.98732";
+
+  src = fetchurl {
+    url = "https://ftp.eiffel.com/pub/download/latest/pp/PorterPackage_${lib.versions.majorMinor version}_rev_${lib.versions.patch version}.tar";
+    sha256 = "sha256-/XoewqCeh1NfB3ve9UL+0WZfZ5DEa4N7REl67Ftlxt0=";
+  };
+
+  nativeBuildInputs =
+    with pkgs;
+    [
+      coreutils
```

Comment:
At least coreutils and sed are part of stdenv iirc, so we should not duplicate them here.